src/hydra-evaluator/hydra-evaluator.cc

```diff
@@ -13,7 +13,57 @@
 
 using namespace nix;
 
-typedef std::pair<std::string, std::string> JobsetName;
+typedef std::pair<std::string, std::string> JobsetSymbolicIdentity;
```

Comment:
I would just keep this as `JobsetName` since "symbolic identity" sounds like a fancy way of saying "name" :-)